appium/webdriver/webdriver.py

```diff
@@ -1285,7 +1285,14 @@ def set_clipboard_text(self, text, label=None):
         :param text: The text to be set
         :param label: Optional label argument, which only works for Android
         """
-        self.set_clipboard(bytes(text.encode('UTF-8')), ClipboardContentType.PLAINTEXT, label)
+
+        def _bytes(value, encoding):
```

Comment:
does it make sense to put this helper method into some separate helper module?

Reply:
I thought so, but nothing method used `bytes` so far. I'd like to extract here in next time.